src/pages/search.tsx

```diff
@@ -58,6 +58,11 @@ const AUTO_SORT_MAP: Record<string, "-"> = {
 }
 
 const PAGE_SIZE = 50
+
+const doesQuerySpecifyFormat = (query: string) : boolean => {
+  return query.includes('legal:') || query.includes('banned:') || query.includes('restricted:') || query.includes('format:') || query.includes('f:');
```

Comment:
Here the query is not meant for scryfall, instead it's meant to be sent to our backend!
As a consequence, the syntax is different. You can find a reference here: https://commanderspellbook.com/syntax-guide/.

Please, change this line to reflect the available legality keywords from the Spellbook query syntax, namely `legal`, `banned`, and `format`, thank you

Reply:
Ah, thanks, I missed that, pushed the fix for it.